std_msgs/Header header
geometry_msgs/Pose2D face
combi_darknet_openface/DetectionResult[] detectionResults
int64 gazedObjectIndex
int64 OUT_OF_VIEW=-1
